src/quest/manager.py

```diff
@@ -151,3 +165,26 @@ async def get_event(self, workflow_id: str, name: str, identity: str | None) ->
     async def get_identity_queue(self, workflow_id: str, name: str, identity: str | None) -> IdentityQueue:
         await self._check_resource(workflow_id, name, identity)
         return self._wrap(IdentityQueue(), workflow_id, name, identity)
+
+    def register_alias(self, alias: str, workflow_id: str):
+        if alias not in self._alias_dictionary:
+            self._alias_dictionary[alias] = workflow_id
+        else:
+            raise Exception('Alias already exists')
+
+    def deregister_alias(self, alias: str):
+        if alias in self._alias_dictionary:
+            del self._alias_dictionary[alias]
+
+
+    # TODO refactor, deduplicate
+    def _check_alias(self, alias: str) -> str | None:
+        """Check to see if an alias exists
+
+        Returns a str of the workflow id if alias exists, None if it doesn't
+        """
+        return self._alias_dictionary.get(alias)
+
+def find_workflow_manager() -> WorkflowManager:
+        if (manager := workflow_manager.get()) is not None:
```

Comment:
Having the same issue here we saw in class. `workflow_manager` exists, but `.get()` returns a LookupError